src/app/submit-item/components/submit-page/submit-page.component.html

```diff
@@ -31,22 +31,25 @@ <h2>Dein Fall</h2>
           </p>
           <div class="field-container">
             <p>Um welchen Medien-Typ geht es?*</p>
-            <mat-form-field class="form-select responsive-100">
-              <mat-select [(ngModel)]="form.item_type_id" name="item_type_id" title="Medientyp" required panelClass="test-panel">
+            <mat-form-field class="form-select responsive-100" appearance="outline">             
+              <mat-select [(ngModel)]="form.item_type_id" name="item_type_id" placeholder="Hier wählen" title="Medientyp" required>               
                 <mat-option *ngFor="let itemType of itemTypes$ | async" [value]="itemType.id">
                   {{ itemType.id | itemType }}
                 </mat-option>
               </mat-select>
             </mat-form-field>
           </div>
           <div class="field-container">
+            <p *ngIf="form.item_type_id === null">
+              Bitte gib hier <span class="display-item-type">einen Link/ Textnachricht oder Aussage</span> ein*
+            </p>
             <p *ngIf="form.item_type_id === '1' ">
             Bitte gib hier <span class="display-item-type">den {{ form.item_type_id | itemType }}</span> ein*
             </p>
             <p *ngIf="form.item_type_id === '2' ">
               Bitte gib hier <span class="display-item-type">eine {{ form.item_type_id | itemType }}</span> ein*
             </p>
-            <textarea class="textarea" rows="5" [(ngModel)]="form.content" name="content" required></textarea>
+            <textarea class="textarea" [(ngModel)]="form.content" rows="5" placeholder="Hier Inhalt eingeben" onfocus="this.placeholder=''" onblur="this.placeholder='Hier Inhalt eingeben'" name="content" required></textarea>
```

Comment:
Außerdem passt die `font-size` des Inhalts nicht zu der des Placeholders, was zu einem "springen" der Seite führt, wenn man in die `textarea` klickt.

Reply:
die fontsize ist dem Umstand geschuldet, dass es auf Seiten mit mehreren Eingabefeldern (eben der Submit Page) gleich aussehen soll: in allen Feldern soll der Placeholder gleiches Aussehen, gleiche Größe etc. haben. Sah vorher komisch aus, wenn der Placeholder die größere Schrift hatte.
Was "springt" denn da? Den Effekt seh ich gar nicht ... ?

Reply:
Wenn du in die Textarea klickst und dann den Cursor wieder aus dem Textfeld bewegst, dann springt der ganze untere Teil der Seite (Illustration, Optionale Angaben, Footer) ein paar Pixel nach oben.

Reply:
jetzt weiß ich, was du meinst.
Das liegt aber nicht an den Placeholdern (hab gerade mal alles entfernt).
Vermute, das sind die unterschiedlichen Rahmenstärken. Die waren auch vorher so - ist aber  keinem aufgefallen.